modules/upgrade/pages/proxy-migration.adoc

```diff
@@ -0,0 +1,185 @@
+[[proxy-migration]]
+= Proxy Migration
+
+// also see client-migration.adoc
+Upgrading from {susemgrproxy} 3.2 with the latest service pack (SP) to version 4 can be automated, but requires some preparation steps.
+
+////
+To upgrade the SP version on SLE{nbsp}12 (for example, upgrading from SLE{nbsp}12 or any SLE{nbsp}12{nbsp}SPx to SLE{nbsp}12{nbsp}SP4) can be fully automated and requires no additional preparation.
+////
+
+== Preliminary Considerations
+
+The recommended order for migrations is to first migrate the {productname} server and then the proxies.
+
+.Using SUSE Manager proxy 3.2 with SUSE Manager 4
+[NOTE]
+====
+A SUSE Manager proxy 3.2 cooperates just fine with a new SUSE Manager 4. After the server has been migrated to version 4, proxies can be autoupgraded from the SUSE Manager server with YaST autoinstallation.
+====
+
+Right now, migrating SUSE Manager proxies 3.2 only works for proxies that have been registered as traditional clients.
+Proxies that are Salt minions cannot be migrated to the new version this way.
+
+
+
+[[proxy-migration-inplace]]
+== Upgrading SUSE Manager Proxy In-place
+
+
+
+[[proxy-migration-inplace-prep]]
+=== Preparation
```

Comment:
I would remove the heading here (to avoid the double heading), the Procedure can stand on its own.

Reply:
agreed, makes sense.